doc/classes/EditorSpinSlider.xml

```diff
@@ -51,4 +51,12 @@
 			</description>
 		</signal>
 	</signals>
+	<theme_items>
+		<theme_item name="updown" data_type="icon" type="Texture2D">
+			Single up/down graphic for both up/down buttons. Likely to be deprecated when individual up/down buttons will be implemented.
```

Comment:
I'm not sure about that deprecation note.

Reply:
No, yeah, if it's not deprecated **now** there is no point.

If I may suggest, but it's a nitpick:
```suggestion
			Single texture representing both the up and down buttons.
```
or
```suggestion
			Single texture combining both the up and down buttons.
```
or similar.